src/Testing/TestResponseMixin.php

```diff
@@ -114,4 +120,108 @@ public function assertGraphQLErrorFree(): \Closure
             return $this;
         };
     }
+
+    public function assertGraphQLSubscriptionAuthorized(): \Closure
+    {
+        return function (TestCase $testClassInstance): TestResponse {
+            Assert::assertTrue(method_exists($testClassInstance, 'postJson'));
+
+            $channel = $this->json('extensions.lighthouse_subscriptions.channel');
+
+            $testClassInstance
+                // @phpstan-ignore-next-line present in \Illuminate\Foundation\Testing\Concerns\MakesHttpRequests
+                ->postJson('graphql/subscriptions/auth', [
+                    'channel_name' => $channel,
+                ])
+                ->assertSuccessful()
+                ->assertExactJson([
+                    'message' => 'ok',
+                ]);
+
+            return $this;
+        };
+    }
+
+    public function assertGraphQLSubscriptionNotAuthorized(): \Closure
+    {
+        return function (TestCase $testClassInstance): TestResponse {
+            Assert::assertTrue(method_exists($testClassInstance, 'postJson'));
```

Comment:
Assert::assertTrue(method_exists($testClassInstance, 'postJson'), 'Expected the given $testClassInstance to use the trait Illuminate\\Foundation\\Testing\\Concerns\\MakesHttpRequests.');